librosa/display.py

```diff
@@ -375,13 +392,13 @@ class FJSFormatter(mplticker.Formatter):
 
     def __init__(
         self,
-        fmin,
-        major=True,
-        unison=None,
-        unicode=True,
-        intervals=None,
-        bins_per_octave=None,
-        n_bins=None,
+        fmin: int,
+        major: bool = True,
+        unison: Optional[str] = None,
+        unicode: bool = True,
+        intervals: Optional[Union[str, Collection[float]]] = None,
```

Comment:
Hrm - this seems incorrect to me (on my part!).  Intervals is not really optional (ditto bins_per_octave, n_bins) - but they're set to None by default here.  I think we should push these up to required, named positional arguments.  This is a new class in 0.10, so no worries about breaking API here.

(Also I forgot to docstring them in the past PR.  Oof!)

Reply:
specshow has to still keep `intervals` as optional, right? So you should probably add validation in `__decorate_axis`. I'd ask you to do this in a follow up PR if possible since I'm not sure exactly what you want here

Reply:
It's optional for specshow, but not for this specific tick formatter.

I think it makes sense to take this in a separate issue, possibly tying in to #1618